tests/files/gce_centos7-flannel-addons.yml

```diff
@@ -7,7 +7,6 @@ mode: ha
 # Deployment settings
 kube_network_plugin: flannel
 helm_enabled: true
-istio_enabled: true
```

Comment:
why delete this fragment?

Reply:
The installed istio is 0.2 which breaks the CI.